sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/components/projectoverview/SampleCount.groovy

```diff
@@ -0,0 +1,20 @@
+package life.qbic.portal.sampletracking.components.projectoverview
+
+import groovy.transform.EqualsAndHashCode
+
+/**
+ * <b>Stores the number of failing and passing sample counts for a status</b>
+ *
+ * @since 1.0.0
+ */
+@EqualsAndHashCode
+class SampleCount {
```

Comment:
Please have a look at the RelativeCount class instead

Reply:
Why? The relative count is showing the relation between passed and total samples. That would make no sense for me 🤔

Reply:
The relative count is what lies behind each cell. It was only the relative count until now but it can be extended by further specifying/replacing the `count` into `passed` and `failed` then you would also have the failed/passed relative number.